packages/maskbook/src/extension/options-page/DashboardRouters/Plugins.tsx

```diff
@@ -0,0 +1,100 @@
+import { useState } from 'react'
+import { TextField, IconButton } from '@material-ui/core'
+import { makeStyles, createStyles, ThemeProvider } from '@material-ui/core/styles'
+import SearchIcon from '@material-ui/icons/Search'
+import ClearIcon from '@material-ui/icons/Clear'
+import PluginCard from '../DashboardComponents/PluginCard'
+import type { Plugin } from '../../../database'
+import DashboardRouterContainer from './Container'
+import { useI18N } from '../../../utils/i18n-next-ui'
+import { extendsTheme } from '../../../utils/theme'
+
+const useStyles = makeStyles((theme) =>
+    createStyles({
+        root: {
+            width: '100%',
+            maxWidth: 360,
+            backgroundColor: theme.palette.background.paper,
+        },
+        pluginList: {
+            padding: 0,
+            margin: 0,
+            paddingTop: theme.spacing(3),
+            display: 'grid',
+            gridTemplateColumns: '1fr 1fr',
+            [theme.breakpoints.down('sm')]: {
+                display: 'block',
+            },
+        },
+        pluginItem: {
+            listStyle: 'none',
+        },
+    }),
+)
+
+const pluginsTheme = extendsTheme((theme) => ({}))
+
+const mockPlugins: Plugin[] = [
+    {
+        id: '1',
```

Comment:
why need id?

Reply:
it refers to the things like `com.mask.redpacket`